messaging/src/main/java/org/frankframework/jms/JmsMessagingSourceFactory.java

```diff
@@ -24,56 +24,53 @@
 
 import jakarta.jms.ConnectionFactory;
 
+import lombok.extern.log4j.Log4j2;
+
 import org.frankframework.configuration.ConfigurationException;
 import org.frankframework.core.IbisException;
 
-
 /**
- * Factory for {@link JmsMessagingSource}s, to share them for JMS Objects that can use the same.
- * <p>
- * JMS related IBIS objects can obtain a MessagingSource from this class. The physical connection is shared
- * between all IBIS objects that have the same connectionFactoryName.
+ * Factory for {@link MessagingSource}s, to share them for JMS Objects that can use the same.
  *
- * @author  Gerrit van Brakel
- * @since   4.4
+ * @author Gerrit van Brakel
  */
-public class JmsMessagingSourceFactory extends AbstractMessagingSourceFactory {
-
+@Log4j2
+public class JmsMessagingSourceFactory {
```

Comment:
Merged the `JmsMessagingSourceFactory` and `AbstractMessagingSourceFactory` as there was only 1 implemention.